### ISSUE TITLE
get_candles returns Nonetype for many coins

### ISSUE THREAD

**Issue by tjohnson310** [ericsomdahl/python-bittrex#163]
Anyone else having trouble with get_candles? I find that when I try to pull a data from a list of coin names, some of the coin names don't return candle profiles, but instead return Nonetype. All thoughts welcome!

**Comment by sigmund505** [comment]
> Anyone else having trouble with get_candles? I find that when I try to pull a data from a list of coin names, some of the coin names don't return candle profiles, but instead return Nonetype. All thoughts welcome!


Yes, I'm also getting:

`TypeError: 'NoneType' object is not subscriptable`
`in fill_missing_candles filled_list = [candles[0]]`

Truly enjoyed Josiah Swaim's excellent article "Guide to Cryptocurrency Arbitrage: How I made 1% profit an hour" and hope to get this demo script running.  Was able to get the Bittrex API_V2 issue to the correct lib install...

**Comment by tjohnson310** [comment]
Yes, it is faulty when you are pulling data for multiple coins at the same time

Sent from my Verizon, Samsung Galaxy smartphone

________________________________

**Comment by tjohnson310** [comment]
Try to limit # of coins to 10 ish

Sent from my Verizon, Samsung Galaxy smartphone

________________________________

**Comment by sigmund505** [comment]
Thanks!!!

**Comment by tjohnson310** [comment]
Would love to collab on trading bots if anyone is open!
________________________________
